packages/builders/scripts/inter-protocol/replace-electorate-core.js

```diff
@@ -0,0 +1,99 @@
+/* global process */
```

Comment:
How about a `/** @file ... */` comment including usage?

By way of example: [init-orca.js](https://github.com/Agoric/dapp-orchestration-basics/blob/c4ea22446ba8f1b7b62a7746d4c3c79d16fb6037/contract/src/builder/init-orca.js#L1-L12)